doc/quick_start.md

```diff
@@ -1,22 +1,17 @@
 # Quick Start
 
-It's quite simple to try out SQLFlow using [Docker](https://docs.docker.com/).
+It's quite simple to try out SQLFlow using [SQLFlowPlayground On the Cloud](https://playground.sqlflow.tech/).
 
-1. Install [Docker Community Edition](https://docs.docker.com/install/).
-1. Run SQLflow by typing the below command:
-   ```bash
-   docker run --name=sqlflow-mysql -d -p 8888:8888 sqlflow/sqlflow:mysql
-   docker run --net=container:sqlflow-mysql -d sqlflow/sqlflow:latest sqlflowserver
-   docker run --net=container:sqlflow-mysql -d sqlflow/sqlflow:jupyter
-   ```
-1. Wait until the docker containers are all running, then access http://localhost:8888 in your Web browser.
-1. Open the a tutorial notebook like `iris-dnn.ipynb` and run the examples.
+1. Click [here](https://playground.sqlflow.tech/) to open our playgorund website.
+1. Login SQLFlow Playground with your GitHub account, just click the authorization button.
+1. Select some project to try out SQLFlow.
```

Comment:
Wait until your notebook is ready then choose a notebook (`.ipynb`) file to tryout SQLFlow, or create a new one.